### PR TITLE
Allow to set the reference commit for the branch list

### LINKED ISSUE #80: How to highlight todos in current branch vs. master?
I would like to display the TODOs in the current branch with `git diff master` rather than the default `git diff HEAD^`. How can I do that?

### PR DESCRIPTION
**Pull request by smaret**
Closes #80.

### CODE REVIEW

File: magit-todos.el [hunk 1]
```diff
@@ -328,14 +328,18 @@ used."
...
+\"git diff magit-todos-branch-list-ref-commit\".
 
 This can be toggled locally in Magit buffers with command
 `magit-todos-toggle-branch-list'."
   :type '(choice (const :tag "Never" nil)
                  (const :tag "In non-master branches" branch)
                  (const :tag "Always" t)))
 
+(defcustom magit-todos-branch-list-ref-commit '"HEAD^"
+  "Reference commit for branch diff to-do list."
```
Comment (alphapapa) [inline magit-todos.el]: I think the docstring should elaborate a bit.  Maybe something like:

```
Commit ref passed to "git diff" command used to make branch diff list.
```

**Comment by smaret** [comment]
Thanks for the review. I've made the changes you suggested.

**Comment by alphapapa** [comment]
Thanks!

**Comment by alphapapa** [comment]
BTW, I made a small mistake when rebasing, but I've fixed it on `master` now.

**Pull request merged**